README.md

```diff
@@ -48,8 +48,6 @@ PHP Mode is available from [package systems provided by several OSs][php-mode-pa
 
 Please consider installing the package provided in `sid` (aka: "unstable", a rolling release) in order to benefit from the latest PHP Mode features and performance improvements.  The latest version can be found here: [`elpa-php-mode`][elpa-php-mode].  Investigate "apt-pinning" to make tracking this update stream automatic.
 
-Also, the `php-elisp` package provided by [Debian 9 (stretch)][php-elisp-stretch] and [Ubuntu 18.10][php-elisp-ubuntu1810] and earlier is [extremely old][issue-430], so **PLEASE DO NOT INSTALL IT**.
-
```

Comment:
Poking around the readme a bit more, I see that [there was a goal of maintaining a support matrix based on Debian's support schedule](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users).

Based on that, replacing this with an "unsupported OSes not supported" warning might be good? e.g.,

```suggestion
Note that there [is no support](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users) for package versions installed on [end-of-life OS versions](https://wiki.debian.org/LTS).
```

Reply:
> Poking around the readme a bit more, I see that [there was a goal of maintaining a support matrix based on Debian's support schedule](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users).
> 
> Based on that, replacing this with an "unsupported OSes not supported" warning might be good? e.g.,

@zonuexe, do you want to maintain this?

@NateEag, from what I can tell that wiki thing is mostly about "what's the minimum version of Emacs that php-mode ought to support".  It doesn't make sense to me to link to it here.  Oldoldstable (currently Debian 10 "buster") has Emacs 26.1, php-mode 1.21.0-1, and the only updates buster will see are for things like security updates and ultra obscure corner case data loss bugs...and those must be backported targeted fixes.

Reply:
@sten0 - yup, you're right - I obviously misread that. Good point, sorry about that.

Reply:
It's been a year since Debian 9 Stretch EOL, so it seems that few people are confused anymore. :+1: